src/ripple/ledger/impl/View.cpp

```diff
@@ -262,6 +210,89 @@ accountFunds (ReadView const& view, AccountID const& id,
     return saFunds;
 }
 
+static std::uint32_t
+computeOwnerCount (std::uint32_t curOwnerCount, std::int32_t adj)
```

Comment:
Minor nit: we replicate this logic in `adjustOwnerCount`. I'd like to see only a single function to handle this.